src/azure-cli-core/azure/cli/core/parser.py

```diff
@@ -142,7 +147,11 @@ def validation_error(self, message):
     def error(self, message):
         telemetry.set_user_fault('parse error: {}'.format(message))
         self._handle_command_package_error(message)
-        return super(AzCliCommandParser, self).error(message)
+
+        args = {'prog': self.prog, 'message': message}
+        logger.error('%(prog)s: error: %(message)s', args)
+        self.print_usage(sys.stderr)
+        self.exit(2)
```

Comment:
About this. Will exit here change the existing workflow (I have no idea, so I'm just asking.)